docs/code/FilterParameters.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+import itk
+import sys
+
+input_filename = sys.argv[1]
+
+image = itk.imread(input_filename, itk.F)
+
+# Pythonic snake case keyword arguments:
+#
+#   number_of_iterations
```

Comment:
Not sure if the blank line before and the multiple white spaces before the argument were intentional or whether you think improve readability.

Reply:
These were added for readability -- I will add another blank line for balance and readability.